src/Components/QuizList/QuizListComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import { useQuiz } from "../../Contexts/quizContext"
+import { Link } from "react-router-dom"
+import "./QuizListComponent.css"
+import { QuizType } from "../../Types/quizContext"
+import "../../softui.css"
+
+
+export function QuizListComponent(){
+
+    const {quiz}:{quiz:QuizType[]} = useQuiz()
```

Comment:
if we have defined the type of quiz in quiz context, do we need to define it again here? wouldn't it know the type already?
do let me know if i am missing something here!